be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -50,8 +50,15 @@ class LakeTabletsChannel : public TabletsChannel {
     void add_chunk(vectorized::Chunk* chunk, const PTabletWriterAddChunkRequest& request,
                    PTabletWriterAddBatchResult* response) override;
 
+    void add_segment(brpc::Controller* cntl, const PTabletWriterAddSegmentRequest* request,
+                     PTabletWriterAddSegmentResult* response, google::protobuf::Closure* done) {
+        response->mutable_status()->set_status_code(TStatusCode::NOT_IMPLEMENTED_ERROR);
+    }
```

Comment:
IMO, we should try to not add a method that will be used, it's normally a signal of bad design.

Reply:
ok, i will remove it.